packages/contracts/src/plugins/token/IMerkleDistributor.sol

```diff
@@ -20,13 +20,6 @@ interface IMerkleDistributor {
     /// @notice The merkle root of the balance tree storing the claims.
     function merkleRoot() external returns (bytes32);
 
-    /// @notice Initializes the plugin.
-    /// @dev This method is required to support [ERC-1822](https://eips.ethereum.org/EIPS/eip-1822).
-    /// @param _dao The IDAO interface of the associated DAO.
-    /// @param _token A mintable [ERC-20](https://eips.ethereum.org/EIPS/eip-20) token.
-    /// @param _merkleRoot The merkle root of the balance tree.
-    function initialize(IDAO _dao, IERC20Upgradeable _token, bytes32 _merkleRoot) external;
```

Comment:
Is there any disablerInitialize there that should be done?

Reply:
We will come back to this.It is already in the constructor of the implementation code in `MerkleDistributor`.